build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o pipefail
+
+# Get a clean checkout
```

Comment:
We should already have a clean checkout when running in CI. If you rename this script to `resolve-conflicts.sh` or similar, it should be more clear it's not supposed to be run locally as a matter of course, and this can be removed.